templates/admin/audit_log/index.html.ep

```diff
@@ -11,7 +11,11 @@
     %= include 'layouts/admin_nav'
 
     <div class="col-sm-10">
-        <h2><%= title %></h2>
+        <h2><%= title %>
+            % if (!$audit_enabled) {
+                - auditing disabled!
```

Comment:
It's part of title, I didn't want to introduce different styles there. I'm useless for front end design ;)